UI/window-basic-main.cpp

```diff
@@ -136,6 +136,11 @@ template<typename T> static void SetOBSRef(QListWidgetItem *item, T &&val)
 
 static void AddExtraModulePaths()
 {
+	string plugins_path = getenv("OBS_PLUGINS_PATH");
+	string plugins_data_path = getenv("OBS_PLUGINS_DATA_PATH");
+	obs_add_module_path(plugins_path.c_str(),
+			    (plugins_data_path + "/%module%").c_str());
```

Comment:
This assumes the environment variables are defined and valid. What happens if they're not?

Reply:
An access violation will occur if a null pointer is assigned/constructed to an std::string.

Reply:
Right, will add a check here, thank you!